css/css-values/urls/empty.html

```diff
@@ -0,0 +1,40 @@
+<!doctype html>
+<title>Empty URLs behaviour</title>
+<link rel=help href=https://drafts.csswg.org/css-values/#url-empty>
+<link rel=help href=https://github.com/w3c/csswg-drafts/issues/2211#issuecomment-365677844>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<style>
+#inline-unquoted {
+    background-image: url();
+}
+
+#inline-quoted {
+    background-image: url("");
+}
+</style>
+<link rel=stylesheet href=support/empty-urls.css>
+<div id="inline-unquoted"></div>
+<div id="inline-quoted"></div>
+<div id="external-unquoted"></div>
+<div id="external-quoted"></div>
+<script>
+const ids = [
+  "inline-unquoted",
+  "inline-quoted",
+  "external-unquoted",
+  "external-quoted"
+];
+
+const inline_url = location.href;
+const external_url = new URL(document.querySelector("link[rel=stylesheet]").href, location.href).href;
+
+for (let id of ids) {
+    test(function() {
+        const el = document.getElementById(id);
+        const expected = id.startsWith("inline-") ? inline_url : external_url;
```

Comment:
I'd note that you could have generated this in more interesting ways, e.g., 

```js
const expected = window[id.replace(/-.*/, "_url")];
```

although whether that's a good idea is an open question...

It's certainly fine as it is.